pkg/sensors/exec/execvemap/execve.go

```diff
@@ -20,4 +20,5 @@ type ExecveValue struct {
 	Pad          uint32                     `align:"pad"`
 	Namespaces   processapi.MsgNamespaces   `align:"ns"`
 	Capabilities processapi.MsgCapabilities `align:"caps"`
+	BinaryPath   processapi.Binary          `align:"binary_path"`
```

Comment:
This should be `align:"bin"` here. You fix that in a next commit but you can have it correct from here.

Reply:
this is fixed indeed, maybe I fixed it in the wrong commit and it's still present in some commits https://github.com/cilium/tetragon/pull/1731/files#diff-52bf8d0069ecc52a261cdf0cb40e9b51da31ec1c19cfda5b520729203c13618dR21. I can rebase if it's the case.

Reply:
Yes, the final result is (and was) fine. What I wanted to say here is that it is good to have separate commits as clean as possible because (1) it is easier to review and (2) it is easier to revert a specific commit rather than reverting a full PR. 

As an example for this specific thing you introduce 
```
BinaryPath processapi.Binary `align:"binary_path"`
```
in the third commit https://github.com/cilium/tetragon/pull/1731/commits/087cd9eabfafb1f568ad6dd72aae86236fc7435c. Then you change `binary_path` to `bin` in the seventh commit https://github.com/cilium/tetragon/pull/1731/commits/b4d1f789ffe3a66756267b8eca4680d52f90a8db. If you see the PR as a whole it is fine, if you see the PR as separate commits it is not. The same applies to other minor things in this PR as well.

I am fine with that in this PR. This is just a comment about future PRs.

Reply:
Ah yeah, it would be nice to fix that to have each commit to compile and run. I might have end up with this situation because I reorganized those commits many times and I might have been wrong in some rebase :/! Thanks